src/components/FriendPage/FriendPage.js

```diff
@@ -26,15 +27,20 @@ const FriendPage = ({ id }) => {
     )
   }
 
-  if (friend) {
+  const deleteFriend = () => {
+    dispatch(removeFriend(friend))
+    setStatus(false)
+  }
+
+  if (friend && active) {
     return (
       <main className='friend-gift-page'>
         <header>
           <img src={getIcon(id)}></img>
```

Comment:
Is there a way we can add an additional button down the road to bring the user back to home once we implement Router?

Reply:
Definitely! Good call.